Backend/auction-app/src/main/java/com/example/auctionapp/service/UserService.java

```diff
@@ -26,6 +26,8 @@
     private BaseRepository<Role> roleRepository;
     private final PasswordEncoder passwordEncoder;
 
+    private final Long USER_ROLE_ID = 2L;
```

Comment:
This property can be `static` as well, as it is constant and not prone to change